### PR TITLE
Replace RECESS with LESS

### PR DESCRIPTION
**Pull request by jw-kaurand**
Replaces RECESS compilation with grunt-contrib-less.  Linting will be added back in in the next PR.

Branching off the previous LESS/Chromeless branch feature/controls-less-splitting-mk2

JW7-4098

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -41,33 +41,15 @@ module.exports = function(grunt) {
...
-            },
-            lint: {
-                files: [{
-                    expand: true,
-                    ext: '.css',
-                    dest: 'bin-debug/skins/',
-                    cwd: 'src/css/',
-                    src: '{,*/}*.less'
-                }]
+                dumpLineNumbers: false,
```
Comment (egreaves) [inline Gruntfile.js]: The default is `false` ([ref](https://github.com/gruntjs/grunt-contrib-less#dumplinenumbers))

File: Gruntfile.js [hunk 2]
```diff
@@ -41,33 +41,15 @@ module.exports = function(grunt) {
...
+                paths: ['src/css', 'src/css/*']
             },
             internal: {
                 options: {
-                    compile: true
+                    dumpLineNumbers: true
```
Comment (egreaves) [inline Gruntfile.js]: This is not a valid option. It should be `'comments'|'mediaquery'|'all'` ([ref](https://github.com/less/less-docs/blob/master/content/usage/using-less-in-the-browser.md#dumplinenumbers))

File: Gruntfile.js [hunk 3]
```diff
@@ -76,7 +58,7 @@ module.exports = function(grunt) {
             },
             debug: {
                 options: {
-                    compile: true
+                    dumpLineNumbers: true
```
Comment (egreaves) [inline Gruntfile.js]: See above

**Comment by jw-kaurand** [comment]
Closed in favor of https://github.com/jwplayer/jwplayer/pull/1951

**Pull request closed without merging**